envs/env.py

```diff
@@ -112,6 +115,13 @@ def __init__(self,
         self.parameter_fetch_error = False
         self._max_duration = max_duration
         self.max_time_steps_per_episode = 500 if self._is_testing else max_time_steps_per_episode
+        
+        #Parameters for random variations link characteristics, bandwidth and latency at the moment
+        self.random_variation_step = int(max_time_steps_per_episode/3) if self._is_testing else np.random.randint(max_time_steps_per_episode/4, 3*max_time_steps_per_episode/4)
```

Comment:
Please use some parameters when you initialize the environment, so that we can set them from the command line if we need and put some default values as we have done with other params. Also the name `random_variation_step` is not very expressive of the scope of that variable. What about `test_variation_start`, `variation_range_start`, `variation_range_end`?  Or something like that instead of hard-coding those divisions.